src/main/java/com/klodnicki/bike/rest/API/Bike/exception/ExceptionHandler.java

```diff
@@ -0,0 +1,14 @@
+package com.klodnicki.bike.rest.API.Bike.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+@RestControllerAdvice
+public class ExceptionHandler {
```

Comment:
Please, change the name of this class as it conflicts with annotation one and creates unneeded confusion.

Reply:
great! thanks for that!